### PR TITLE
Feature/categorizedentries

### PR DESCRIPTION
**Pull request by marcos-de-barros-vioque**
Hi!
With this PR I'm filtering (and mapping) the form input data in the `CategoryOverview` component and imported it to the category pages (logic = when the user selects category "Food" in the form, the log entry should be displayed on the "/food" page). 
In addition, the visitor gets a success response after having clicked on the "Submit" button.
I've also added a function to the router.push in the `NewSpendingForm` component, so that the user is redirected to the corresponding category page depending on the selected category.
Thanks

### CODE REVIEW

File: components/CategoryOverview.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
+import {useRouter} from "next/router";
+
+export default function CategoryOverview({spendingInput, categoryName}) {
+  const routing = useRouter();
+  console.log({spendingInput});
```
Comment (stefankorder) [inline components/CategoryOverview.js:6]: Please delete this `console.log`.

File: components/NewSpendingForm.js [hunk 2]
```diff
@@ -11,7 +11,10 @@ export default function NewSpendingForm({onAddSpending}) {
     const data = Object.fromEntries(formData);
 
     onAddSpending(data);
-    router.push("./spendings");
+
+    const {category} = data;
+    alert("Your spending has been successfully added!");
```
Comment (stefankorder) [inline components/NewSpendingForm.js:16]: If possible don't use alert. Better bulid a modal to give the user your feedback.

**Review by stefankorder — APPROVED** [review]
Good work! Just 2 recommends.

File: components/NewSpendingForm.js (hunk 2 above)
Comment (marcos-de-barros-vioque) [reply components/NewSpendingForm.js:16]: Good point, Stefan. I'll try to add it to my next PR. Thanks a bunch for the feedback!

**Pull request merged**